homeassistant/components/dialogflow/__init__.py

```diff
@@ -103,6 +103,8 @@ def get_api_version(message):
     if message.get("responseId") is not None:
         return V2
 
+    raise ValueError(f"Unable to extract API version from message: {message}")
+
```

Comment:
Small change in behavior here - as previously it would implicitely return `None`.
I think it makes more sense here to raise an error - but happy to adjust